controllers/drplacementcontrol_controller.go

```diff
@@ -233,21 +234,36 @@ func (r *DRPlacementControlReconciler) Reconcile(ctx context.Context, req ctrl.R
 		return ctrl.Result{}, err
 	}
 
+	schedule, err := r.getSchedule(ctx, drpc)
```

Comment:
DRPolicy object will be saved in the DRPCInstance.  I think you should use it (not yet merged), that way, you will not have to get the DRPolicy multiple times, one for the Schedule and another time for the ReplicationClassSelector